DEVELOPMENT_ENVIRONMENT.md

```diff
@@ -29,45 +48,40 @@
 	- If you've cloned the repository from the command line or a Git client, you obviously only need to perform the last step
 - Install [Payara Tools](https://marketplace.eclipse.org/content/payara-tools)
 - Install the [Vaadin Plugin for Eclipse](https://marketplace.eclipse.org/content/vaadin-plugin-eclipse) (no need to install the commercial UI designer)
-- Add a Payara server to Eclipse and enter the credentials you specified when setting up the server
+- Add a Payara server to Eclipse 
+    - if you've chosen the setup based on [SERVER_SETUP](SERVER_SETUP.md), you'll have to adapt the following paths accordingly
```

Comment:
I'd rather directly include the alternative default paths below (I have added comments accordingly).